tests/unit/CapabilityTest.php

```diff
@@ -0,0 +1,65 @@
+<?php
+namespace Automattic\LegacyRedirector\Unit\Tests;
+
+use Automattic\LegacyRedirector\Capability;
+use Brain\Monkey\Functions;
+use Brain\Monkey;
+use Yoast\WPTestUtils\BrainMonkey\TestCase;
+
+/**
+ * Capability Class Unit Test
+ */
+class CapabilityUnitTest extends TestCase {
+
+	/**
+	 * Setup before each test
+	 *
+	 * @return void
+	 */
+	protected function setUp() {
+		parent::setUp();
+		Monkey\setUp();
+	}
+
+	/**
+	 * TEar down after each test
+	 *
+	 * @return void
+	 */
+	protected function tearDown() {
+		Monkey\tearDown();
+		parent::tearDown();
+	}
```

Comment:
Because this class extends `Yoast\WPTestUtils\BrainMonkey\TestCase` then I believe these Monkey setup and teardown is already handled - see https://github.com/Yoast/wp-test-utils/blob/develop/src/BrainMonkey/TestCase.php

Reply:
even better, i've removed that now.